src/components/shadows/examples/surface-shadows-bad-usage.tsx

```diff
@@ -0,0 +1,85 @@
+import { Component, h } from '@stencil/core';
+
+@Component({
+    tag: 'limel-example-shadows-bad-usage',
+    shadow: true,
+    styleUrl: 'surface-shadows-bad-usage.scss',
+})
+export class SurfaceShadowBadUsageExample {
+    public render() {
+        return [
+            <div class="surface-shadows-bad-usage">
+                <div class="do-not">
+                    <limel-header
+                        icon="brake_warning"
+                        heading="Don't"
+                    ></limel-header>
+                    <div class="content">
+                        <p>
+                            Lorem Ipsum racconta una storia. Non come le altre.
+                            Un individuo che, in un giorno d'estate, s'imbatte
+                            in qualcosa di diverso dalla sua consuetudine. Egli
+                            intreccia la sua vita con quelle di altre persone
+                            del suo paese. Persone che hanno problemi di cuore,
+                            persone che, semplicemente, vorrebbero qualcuno con
+                            cui parlare.
+                        </p>
+                        <div class="box">
+                            <p>To look distinct, I only use a shadow effect.</p>
+                            <p>
+                                But that makes me look like a temporay popover
+                                or some sort of message. User may think I am
+                                covering some of the content.
+                            </p>
```

Comment:
```suggestion
                            <p>To look distinct, I only use a shadow effect.</p>
                            <p>
                                But that makes me look like a temporary popover
                                or some sort of message. The user might think I am
                                covering some of the content.
                            </p>
```

Just fixing the typos ☝️ _or_ 👇

I _LOVE_ these do:s and don't:s, and while I kind of like the personal feeling of the elements speaking in first person to the user, I think it might be better for clarity to keep the "don't" and "do" form throughout the text, like this:

```suggestion
                            <p>Do not use only a drop-shadow to make an element distinct.</p>
                            <p>
                            	Doing so makes the element look like a temporary
                            	popover, or some sort of message. The user might
                            	think it is covering some of the content.
                            </p>
```

Reply:
Tnx :) In my head, the first person talking makes the example more clear and more understandable. "Look, I'm the bad example! Don't be like me." :P I think it's more pedagogic, and has less authoritative tone.

But I have no emotional attachment to it. So you can merge which ever you want @adrianschmidt